src/main/java/com/amihaiemil/camel/RtYamlLines.java

```diff
@@ -54,7 +52,23 @@
 
     @Override
     public YamlLines nested(final int after) {
-        return null;
+        final List<YamlLine> nestedLines = new ArrayList<YamlLine>();
+        YamlLine start = null;
+        int index = 0;
```

Comment:
@SherifWaly every line knows its number. So you can replace this ``index`` variable with ``line.number()``, no? :D and in the for you would have:

```java
if(line.number() == after) {
    start = line;
}
if(line.number() > after) {
    if(line.indentation() > start.indentation()) {
        nestedLines.add(line);   
    } else {
        break;
    }
}
```